packages/itwinui-css/src/style/variables.scss

```diff
@@ -2,7 +2,7 @@
 // See LICENSE.md in the project root for license terms and full copyright notice.
```

Comment:
Everything in `styles` are variables, maybe it makes sense to rename this file to something more specific?

Reply:
Do you think doing something like 'misc-variables' is better, or to separate the variables out into more distinct groups like 'component-padding' and 'component-height'? I couldn't think of anything concise that represented all of the variables in this file.

Reply:
- rename to `component.scss`
- move everything to `global.scss`

Reply:
Thanks, done